Vagrantfile

```diff
@@ -8,11 +21,14 @@ Vagrant.configure("2") do |config|
     config.cache.enable :npm
   end
 
+  config.vm.box = vagrant_box
   config.vm.hostname = "girder"
+  config.vm.provider "virtualbox" do |virtualbox|
+    virtualbox.memory = 2048
+  end
 
-  # Disable port forwarding for ansible testing, since it makes running
-  # VMs in parallel harder.
-  unless ENV["ANSIBLE_TESTING"]
+  # Disable port forwarding for ansible testing, since it makes running VMs in parallel harder.
+  unless is_testing
     config.vm.network "forwarded_port", guest: 8080, host: 9080
```

Comment:
Yes, I had the exact same experience a long time ago, and there wasn't a solution at the time. We should file an issue and revisit it though.